chpl/chpl-service/src/main/java/gov/healthit/chpl/dto/CertifiedProductDTO.java

```diff
@@ -80,6 +80,14 @@
      */
     public static final int CERTIFIED_DATE_CODE_LENGTH = 6;
 
+    /**
+     * REGEX that matches a CHPL Product ID for searching.
+     * Requires first four components (Edition, ATL, ACB, Developer Code).
+     * Optional for remaining parts.
+     */
+    public static final String CHPL_PRODUCT_ID_SEARCH_REGEX =
```

Comment:
I'm not sure about this (or if it matters) - Shouldn't this be named CHPL_PRODUCT_NUMBER_SEARCH_REGEX?